src/hooks/useRedirectHook.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from "react"
+import { Redirect, useHistory } from "react-router-dom"
```

Comment:
unused imports `React` and `Redirect`

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/359/commits/b910e4654fdd2a7fc17860197412921b364d5be3